chrome/content/zotero/preferences/preferences_export.js

```diff
@@ -197,17 +219,21 @@ Zotero_Preferences.Export = {
 			var treerow = document.createElement('treerow');
 			var domainCell = document.createElement('treecell');
 			var formatCell = document.createElement('treecell');
+			var localeCell = document.createElement('treecell');
 			var HTMLCell = document.createElement('treecell');
```

Comment:
Sorry, I know this isn't your code, but I saw it and now it's bothering me. Could you change this (and below) to `htmlCell`